src/plugins/foxPage/components/FoxTab.stories.tsx

```diff
@@ -27,7 +27,7 @@ export const FoxPageTabs: Story = () => (
           fiatAmount={'6000'}
           icons={[
             'https://assets.coincap.io/assets/icons/fox@2x.png',
-            'https://raw.githubusercontent.com/shapeshift/lib/main/packages/asset-service/src/generateAssetData/ethTokens/icons/foxy-icon.png',
+            'https://raw.githubusercontent.com/shapeshift/lib/main/packages/asset-service/src/generateAssetData/ethereum/icons/foxy-icon.png',
```

Comment:
@NeOMakinG Let's ditch the stories altogether in a follow-up PR unless you want to keep them for any reason

Reply:
Alright